Libraries/src/Amazon.Lambda.Annotations.SourceGenerator/Generator.cs

```diff
@@ -162,9 +162,9 @@ public void Execute(GeneratorExecutionContext context)
                         }
                     }
                     
-                    var serializerString = GetSerializerAttribute(context, lambdaMethodModel);
+                    var serializerInfo = GetSerializerInfoAttribute(context, lambdaMethodModel);
```

Comment:
This was changed to `GetSerializerInfoAttribute` because now we are getting the optional `JsonSerializerContext` as well as the serializer name.